conjureplugin/backcompat.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2018 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conjureplugin
+
+import (
+	"fmt"
+	"github.com/palantir/godel-conjure-plugin/v5/backcompat-cli-bundler/conjurebackcompatcli"
+	"github.com/pkg/errors"
+	"io"
+)
+
+func BackCompat(params ConjureProjectParams, projectDir, groupFlagVal, repositoryNameFlagVal, artifactoryUrlFlagVal string, stdout io.Writer) error {
+	k := 0
+	for key, currParam := range params.Params {
+		currentIRBytes, err := currParam.IRProvider.IRBytes()
+		if err != nil {
+			return err
+		}
+
+		//previousVersion, err := getLastTag(projectDir)
```

Comment:
I was trying to reuse the publish logic to determine the path but seems like much of that logic lives in distgo.artifactoryPublisher and there is not a great path to use that.